collector/log/src/main/cfg/org.apache.karaf.decanter.collector.log.cfg

```diff
@@ -30,6 +30,13 @@ ignored.categories=org.apache.karaf.decanter.appender.*
 #fields.add.eventUUID=UUID
 #fields.add.serverTimestamp=TIMESTAMP
 #fields.rename.MDC=customInfo
-#fields.rename.level=severity
+#fields.rename.level=sevetherity
```

Comment:
I guess this change is unexpected right ?

Reply:
Oops. Yes.

Reply:
I undid the accidental change. Sorry about that.